tests/common.py

```diff
@@ -14,7 +14,7 @@
 
 TEST_CONN = os.getenv('SQLA_TEST_CONN', 'postgresql:///cfdm_unit_test')
 
-rest.app.config['NPLUSONE_RAISE'] = True
+rest.app.config['NPLUSONE_RAISE'] = False
```

Comment:
Actually before I revert, the reason why I had shut this off in the first place is because I seeing a bunch of exceptions raised while testing.  It seems all of the errors are being raised around `BaseConcreteCandidate.flags`.  I thought we addressed these already?